js/storage.js

```diff
@@ -0,0 +1,22 @@
+const formName = document.getElementById('username');
+const formText = document.getElementById('message');
+const mail = document.getElementById('email')
+const submitBtn = document.getElementById('submit-btn');
+
+submitBtn.addEventListener('click', () => {
```

Comment:
- [ ] [Optional] After filling the form, and not submitting it, the entered values should be saved in the browser, please consider targeting the ```form``` instead of the ```submit button``` and using the Event listener ```input``` instead of ```click``` so the data could be saved when the user enter it in the form